app/src/main/java/com/github/sdp/ratemyepfl/database/FirestoreItem.kt

```diff
@@ -0,0 +1,17 @@
+package com.github.sdp.ratemyepfl.database
+
+import com.google.firebase.firestore.DocumentSnapshot
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.jsonArray
+import kotlinx.serialization.json.jsonObject
+
+interface FirestoreItem {
```

Comment:
You should rename it to something like RepositoryItem, otherwise it leaks the fact that we use Firestore in our implementation

Reply:
Yes, you are right! I'll fix this thank you.